lib/siwapp/invoices.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Siwapp.Invoices do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "invoices" do
+    field :identification, :string
+    field :name, :string
+    field :email, :string
+    field :contact_person, :string
+    field :net_amount, :integer, default: 0
+    field :gross_amount, :integer, default: 0
+    field :paid_amount, :integer, default: 0
+    field :draft, :boolean, default: false
+    field :paid, :boolean, default: false
+    field :sent_by_email, :boolean, default: false
+    field :number, :integer
+    field :issue_date, :date
+    field :due_date, :date
+    field :failed, :boolean, default: false
+    field :deleted_number, :integer
+    field :currency, :string
+    field :invoicing_address, :string
+    field :shipping_address, :string
+    field :notes, :string
+    field :terms, :string
+    field :deleted_at, :utc_datetime_usec
+    field :meta_attributes, :map
+    field :series_id, :integer
+    field :customers_id, :integer, null: false
```

Comment:
Correct me if I'm wrong, but here don't you need to indicate the relationship one to many with customers by `belongs_to`?

Reply:
I see that you do that with Taxes and Items in other file, so I see that it's something you'll do later